framework/src/problems/FEProblemBase.C

```diff
@@ -3467,9 +3467,14 @@ FEProblemBase::execute(const ExecFlagType & exec_type)
   // Post-aux UserObjects
   computeUserObjects(exec_type, Moose::POST_AUX);
 
-  // Controls
   if (exec_type != EXEC_INITIAL)
+  {
+    // Pre-ic UserObjects should execute along with the Post-aux group except on EXEC_INITIAL
+    computeUserObjects(exec_type, Moose::PRE_IC);
+
+    // Controls
```

Comment:
This is a useless comment - might as well remove it here.

Reply:
specifically the `//controls` comment or both the comments here?

Reply:
`// Controls`.  Whenever you are modifying code, feel free to remove useless cruft or outdated/incorrect comments.  They are hiding all around moose.